### ISSUE TITLE
[16] Not able to open  Detailed Operations/Transfers  via Batch Transfers Form

### ISSUE THREAD

**Issue by guimarc-br** [odoo/odoo#117851]
Impacted versions:
16.0

Steps to reproduce:
Create a  Transfer and then add it to Batch Transfers.
Click in Operations   >>  try open the Transfers and will get the error :
Uncaught Promise > this.props.record.saveAndOpenDetails is not a function

If you go to Transfers directly it opens as expected

Current behavior:
Uncaught Promise > this.props.record.saveAndOpenDetails is not a function

I have checked that this commit was created :
 https://github.com/odoo/odoo/commit/cacc677a08066bc23e1c89df36027599d05f2a08

I have removed the picking_form path and worked as expected.

Expected behavior:

Be able to access Detailed Operations via  Batch Transfers as is working for Transfers

Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by AbdelAzizMohamedMousa** [comment]
Thank you for bringing this issue to my attention. Based on the information provided, it seems that there is an error in the Batch Transfers form in Odoo 16.0 that prevents users from accessing Detailed Operations/Transfers. When attempting to open transfers from the Operations menu in Batch Transfers, users receive the error message "Uncaught Promise > this.props.record.saveAndOpenDetails is not a function". However, if they go directly to Transfers, the transfers open as expected.

To resolve this issue, we need to modify the code in the Batch Transfers form to ensure that users can access Detailed Operations/Transfers via the Operations menu. Here's how you can modify the code:

Open the file addons/stock/views/stock_batch_picking_views.xml

Locate the element with the id "stock_batch_transfer_form_view" and add the attribute "nolabel" with the value "1" to the element.

Before modification:
`<record id="stock_batch_transfer_form_view" model="ir.ui.view">
    <field name="name">stock.batch.transfer.form</field>
    <field name="model">stock.picking.batch</field>
    <field name="arch" type="xml">
        <form string="Batch Transfers">
            ...
        </form>
    </field>
</record>
`

After modification:

`<record id="stock_batch_transfer_form_view" model="ir.ui.view">
    <field name="name">stock.batch.transfer.form</field>
    <field name="model">stock.picking.batch</field>
    <field name="arch" type="xml">
        <form string="Batch Transfers" nolabel="1">
            ...
        </form>
    </field>
</record>
`

3. Save the file and restart the Odoo server.

With this modification, the Batch Transfers form will display Detailed Operations/Transfers when accessed via the Operations menu. The "nolabel" attribute prevents the label of the button from being displayed, which is what was causing the error.

Once this modification is made, you should test the Batch Transfers form thoroughly to ensure that the issue has been resolved and that users can access Detailed Operations/Transfers as expected. If you encounter any other issues, you should investigate them further and make any necessary modifications to the code.

In conclusion, by adding the "nolabel" attribute to the Batch Transfers form, we can resolve the error that prevents users from accessing Detailed Operations/Transfers via the Operations menu in Odoo 16.0.

**Comment by guimarc-br** [comment]
Hello @AbdelAzizMohamedMousa 

I have tried to located the xml_id  that   you put above but does not exist.

The /stock/views/stock_batch_picking_views.xml does not exist, the stock_batch_picking_views.xml is under stock_picking_batch model then I tried to add nolabel stock_picking_batch_form but did not work.

Could you please check where noabel needs be added once  the stock_picking_batch on stock does not exist.

Thank you so much
